bluesky/callbacks/mpl_plotting.py

```diff
@@ -317,7 +323,7 @@ class LiveGrid(CallbackBase):
     def __init__(self, raster_shape, I, *,
                  clim=None, cmap='viridis',
                  xlabel='x', ylabel='y', extent=None, aspect='equal',
-                 ax=None):
+                 ax=None, axes_positive = ['right', 'up']):
```

Comment:
We avoid mutable defaults except in special circumstances. This should be implemented as a `tuple` -- or maybe as two separate parameters?

Reply:
I am happy to make this a tuple. Perhaps a named tuple is a better option? what are everyones thoughts on named tuples?

Reply:
A namedtuple won't have much impact here, the user won't be able to access it (except via `inspect.signature`, which would be a very special case). A tuple is fine.

Reply:
@awalter-bnl, once the tuple is implemented, we will be ready to merge it after a small "git surgery" :)

Reply:
Would it be better to do this as two parameters so it is easier to set just one of them and let the other be the default?  This would also get is out of the mutable default issue.

Reply:
I am getting to it @mrakitin hopefully by the end of the week !-)